pkgs/by-name/et/eta/package.nix

```
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eta";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "aioobe";
+    repo = "eta";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-UQ8ZoxFAy5dKtXTLwPolPMd7YJeEjsK639RkGCMY6rU=";
+  };
+
```

Have you tried

```suggestion

  outputs = [ "out" "man" ];

```

?

---

Replacing with flag? if yes then it would result in error

---

<details>
<summary>Treeview of result without change</summary>
<pre>
result
├── bin
│   └── eta
└── share
    └── man
        └── man1
            └── eta.1.gz
</pre>
</details>
<details>
<summary>Treeview of result added outputs without removing flags</summary>
<pre>
result
└── bin
    └── eta
</pre>
</details>
</details>
<details>
  <summary>Without Flag</summary>
<pre>
error: builder for '/nix/store/vcvmq35dsfb05a0sgx429rlm0gr0pcvx-eta-1.0.1.drv' failed with exit code 2;
       last 10 log lines:
       > gcc -Wall  -Isrc -c src/options.c  -o obj/options.o
       > mkdir -p obj
       > gcc -Wall  -Isrc -c src/printing.c  -o obj/printing.o
       > gcc -Wall  obj/eta.o obj/eta_calc.o obj/measurements.o obj/options.o obj/printing.o  -o eta
       > Running phase: installPhase
       > install flags: SHELL=/nix/store/306znyj77fv49kwnkpxmb0j2znqpa8bj-bash-5.2p26/bin/bash pkgconfigdir=/nix/store/n9hsl2rvz8gcdgf1cnnyj809xy4iayvq-eta-1.0.1/lib/pkgconfig m4datadir=/nix/store/n9hsl2rvz8gcdgf1cnnyj809xy4iayvq-eta-1.0.1/share/aclocal aclocaldir=/nix/store/n9hsl2rvz8gcdgf1cnnyj809xy4iayvq-eta-1.0.1/share/aclocal install
       > install -D eta \
       >         /usr/local/bin/eta
       > install: cannot create directory '/usr': Permission denied
       > make: *** [Makefile:22: install] Error 1
       For full logs, run 'nix log /nix/store/vcvmq35dsfb05a0sgx429rlm0gr0pcvx-eta-1.0.1.drv'.
</pre>
</details>